edx_proctoring/api.py

```diff
@@ -1526,6 +1526,23 @@ def _calculate_allowed_mins(due_datetime, allowed_mins):
     return actual_allowed_mins, is_exam_past_due_date
 
 
+def _is_self_paced_course(user_id, course_id):
```

Comment:
@schenedx Ah, nevermind. Now I see that we were using the `due_date` from the exam object and not from the context. So based on your knowledge of self-paced courses, will the `due_date` field passed in the context object always be None for a self-paced course, regardless of what someone has set on the subsection in Studio?